src/app/components/AllowanceMenu/index.tsx

```diff
@@ -97,13 +100,13 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
         closeTimeoutMS={200}
         isOpen={modalIsOpen}
         onRequestClose={closeModal}
-        contentLabel="Allowance Options"
+        contentLabel={t("options_label")}
```

Comment:
I always feel using `_` as a separator of "name" and "type" feels off. Feels cleaner to do `options.label`. In this case it's easier to extend it in the future if needed. What do you think?